From: Knud Baastrup <kba@deif.com>
Date: Fri, 24 Oct 2014 13:59:28 +0200

Reduced printing to avoid syslog spam.

Removed unecessary printf in lib/master.c and printk in ioctl.c to avoid
syslog spam. If ECAT_GetSlaveInfo in system SW is called on a slave that
does not exist you get a printf to stderr and printk in syslog. This is
however done every 10ms on each IO card in some applications. The
already returned error code is enough for proper error handling.

diff --git a/lib/master.c b/lib/master.c
--- a/lib/master.c
+++ b/lib/master.c
@@ -266,8 +266,6 @@ int ecrt_master_get_slave(ec_master_t *m
 
     ret = ioctl(master->fd, EC_IOCTL_SLAVE, &data);
     if (EC_IOCTL_IS_ERROR(ret)) {
-        EC_PRINT_ERR("Failed to get slave info: %s\n",
-                strerror(EC_IOCTL_ERRNO(ret)));
         return -EC_IOCTL_ERRNO(ret);
     }
 
diff --git a/master/ioctl.c b/master/ioctl.c
--- a/master/ioctl.c
+++ b/master/ioctl.c
@@ -216,7 +216,7 @@ static ATTRIBUTES int ec_ioctl_slave(
     if (!(slave = ec_master_find_slave_const(
                     master, 0, data.position))) {
         up(&master->master_sem);
-        EC_MASTER_ERR(master, "Slave %u does not exist!\n", data.position);
+        EC_MASTER_DBG(master, 1, "Slave %u does not exist!\n", data.position);
         return -EINVAL;
     }
 
